File: ant-sdk/antd-dart/example/01_connect.dart
import 'package:antd/antd.dart';

/// Demonstrates connecting to the antd daemon and checking health.
void main() async {
  final client = AntdClient();

  try {
    final health = await client.health();
    print('Daemon healthy: ${health.ok}');
    print('Network: ${health.network}');
  } on AntdError catch (e) {
    print('Error: $e');
  } finally {
    client.close();
  }
}

File: ant-sdk/antd-rust/examples/01-connect.rs
use antd_client::{Client, DEFAULT_BASE_URL};

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let client = Client::new(DEFAULT_BASE_URL);

    let health = client.health().await?;
    println!("Daemon healthy: {}", health.ok);
    println!("Network: {}", health.network);

    Ok(())
}
